packages/next-server/lib/amp.ts

```diff
@@ -1,28 +1,28 @@
 import React from 'react'
-import { AmpModeContext } from './amphtml-context'
+import { AmpStateContext } from './amp-context'
 
-export function isAmp({
-  enabled = false,
+export function isInAmpMode({
+  ampFirst = false,
   hybrid = false,
   hasQuery = false,
 } = {}) {
-  return enabled && (!hybrid || (hybrid && hasQuery))
+  return ampFirst || (hybrid && hasQuery)
 }
 
 export function useAmp() {
-  const ampMode = React.useContext(AmpModeContext)
+  const ampState = React.useContext(AmpStateContext)
   // un-comment below to not be considered AMP in dirty mode
```

Comment:
This comment can go away now right? There's no such thing as dirty mode anymore.
```suggestion
```